Extension/src/LanguageServer/client.ts

```diff
@@ -2903,6 +2967,60 @@ export class DefaultClient implements Client {
         this.handleRemoveCodeAnalysisProblems(false, identifiersAndUris);
     }
 
+    public async handleCreateDeclarationOrDefinition(): Promise<void> {
+        let range: vscode.Range | undefined;
+        let uri: vscode.Uri| undefined;
+        // range is based on the cursor position.
+        const editor: vscode.TextEditor | undefined = vscode.window.activeTextEditor;
+        if (editor) {
+            uri = editor.document.uri;
+            if (editor.selection.isEmpty) {
+                range = new vscode.Range(editor.selection.active, editor.selection.active);
```

Comment:
I'm confused by this. If the selection is empty is there supposed to be an active one? You may want to consider using editor.selection.start, editor.selection.end (or something like that instead of active).

Reply:
If `isEmpty` is true, then `start` and `end` position are equal or it means there is an active one but there's no range, just one position. So either `start` and `end` position works or `active` works for this case.